tools/e2etesting/steps/deploytestresources.yml

```diff
@@ -25,6 +25,7 @@ steps:
     scriptPath: '$(BasePath)\DeployEdge.ps1'
     scriptArguments: >
       -ResourceGroupName "$(ResourceGroupName)"
+      -KeysPath "$(KeysPath)"
```

Comment:
See above

Reply:
changed to use $(Agent.TempDirectory) , which is by definition cleaned after each build